pyatlan/model/assets/process.py

```diff
@@ -190,6 +195,16 @@ def airflow_tasks(self, airflow_tasks: Optional[List[AirflowTask]]):
             self.attributes = self.Attributes()
         self.attributes.airflow_tasks = airflow_tasks
 
+    @property
+    def alteryx_workflow(self) -> Optional[AlteryxWorkflow]:
+        return None if self.attributes is None else self.attributes.alteryx_workflow
+
+    @alteryx_workflow.setter
+    def alteryx_workflow(self, alteryx_workflow: Optional[AlteryxWorkflow]):
+        if self.attributes is None:
+            self.attributes = self.Attributes()
+        self.attributes.alteryx_workflow = alteryx_workflow
+
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.